Mathlib/Algebra/Ring/Int/Basic.lean

```diff
@@ -3,82 +3,22 @@ Copyright (c) 2016 Jeremy Avigad. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jeremy Avigad
 -/
-import Mathlib.Algebra.CharZero.Defs
-import Mathlib.Algebra.Group.Int
 import Mathlib.Algebra.Ring.Parity
+import Mathlib.Algebra.Ring.Int.Defs
 
 /-!
-# The integers are a ring
+# Miscellaneous lemmas on the ring structure for `ℤ`.
```

Comment:
It seems like we've got unit and parity lemmas left in this file. Have you considered splitting in another two files?

Reply:
From the files I looked at, the split doesn't seem to help much but I'll try it out right now.

Reply:
Turns out further splitting does help a bit, thanks for the suggestion!